internal/handler/validation/skip.go

```diff
@@ -104,6 +104,10 @@ func getCachedDigest(
 ) (cachedDigest string, cachedError, err error) {
 	cache := ctx.Value(constants.Cache).(caching.Cacher)
 
+	if cache.Ping(ctx) != nil {
```

Comment:
Doesn't this just add latency in the usual case where everything works as intended? If cache isnt up, cache.Get will fail anyways and we have the same behaviour (though with different error)

Reply:
the thing is, i want to a a regular warning message that says the cahc can't be reached. currently, if the `cache.Get` doesn't go though, the error is only logged on debug level (which is intended). only warn if the ping didn't work once during initialization wont work, as connaisseur starts up faster then redis ...

compromise would be: I only ping when the Get returned an error. then i either had a cache miss, or redis wasn't reachable in the first place, so i recheck with the ping which one it was, okey?

Reply:
Sounds better 👍 
However, wouldn't the error message already give away whether the connection worked? i.e. couldn't you parse that instead of doing another roundtrip upon failure?

Reply:
If you only want to have debug level logging of that error anyway, you could also say "if debug, then ping", which would also make prod deployments not waste a roundtrip